analyses/interaction-plots/scripts/03-plot_interactions.R

```diff
@@ -205,32 +206,21 @@ disease_df <-
   readr::read_tsv(disease_file, col_types = readr::cols()) %>%
   dplyr::mutate(gene = factor(gene, levels = genes))
 
-# Previously calculated top 10
-# display_diseases <- disease_df %>%
-#   dplyr::select(disease, mutant_samples) %>%
-#   dplyr::arrange(desc(mutant_samples)) %>%
-#   dplyr::select(disease) %>%
-#   unique() %>%
-#   head(10) %>% # top 10 diseases with highest mutated samples
-#   dplyr::pull(disease)
-
-# Now a manual version
-display_diseases <- c(
-  "Diffuse midline glioma",
-  "Low-grade glioma astrocytoma",
-  "Craniopharyngioma",
-  "High-grade glioma astrocytoma",
-  "Ganglioglioma",
-  "Medulloblastoma",
-  "Meningioma",
-  "Ependymoma",
-  "Schwannoma",
-  "Dysembryoplastic neuroepithelial tumor"
-)
+# Calculate top 10 mutated cancer groups for display
+ display_diseases <- disease_df %>%
+   # remove other from top 10 possibilities
+   dplyr::filter(disease != "Other") %>%
+   dplyr::select(disease, mutant_samples) %>%
+   dplyr::arrange(desc(mutant_samples)) %>%
+   dplyr::select(disease) %>%
+   unique() %>%
+   head(10) %>% # top 10 diseases with highest mutated samples
+   dplyr::pull(disease)
+ 
+ # Print
+ print(display_diseases)
```

Comment:
I don't think that using different cancer groups for different analyses is a problem _per se_. Different analyses will naturally highlight different aspects of the biology in a diverse set of samples like this.  I can see why for the mutational signatures, which shares a figure, we might want to highlight the same cancer types, but even there I am not sure it is strictly required.

However, whatever we decide about "Other HGG" & "Other LGG" etc. we will want to be sure to apply the same logic wherever we use those terms, which may require some careful work/custom code to collapse "missing" groups into those umbrellas. I used analogy in a chat that I think pretty decently gets across my point. If in one place we talked about "mammals, birds, and other animals" and in another place "mammals, birds, reptiles, and other animals", I don't think anyone would be confused, but they _would_ expect reptiles to be included in "other animals" in the first case.

Reply:
@jashapiro I updated all of this using the most recent cancer group displays, however, the legend still has borders and I do not see a way to control that without removing the border from the barplots. Maybe you can take a look, otherwise, we may have to plot the legend separately as @sjspielman has done a few times, and add it back in AI.